### DISCUSSION TITLE
Inovelli Switch Entities available?

### DISCUSSION THREAD

**Discussion by kuntman** [jason0x43/hacs-hubitat#228] [General]
Im trying to merge from hubitat to HA one automation at a time. I got the devices into HA using this, but the inovelli zwave switches only report on/off and energy use. They arent pulling any of the Pushed/Held entity data which is used for scene control. Any changes I can make on my side or is this just a limitation of the API. 

Thanks!

**Comment by jason0x43** [comment]
If they provide button capabilities from Hubitat, there won't be button entities, but they should work as device triggers. The integration handles buttons a bit differently from typical entities; they're event emitters rather than stateful entities.
